recipes/_install.rb

```diff
@@ -1,13 +1,17 @@
+set_broker_id
+set_zookeeper_connect
 
 include_recipe "java"
 
 # setup confluent group
 group node["confluent"]["group"] do
+  gid node['confluent']['uid'] if node['confluent']['uid']
```

Comment:
It would probably be better to be gid.  I usually set the group and user to the same number.